internal/cmd/marketplace/apply/apply.go

```diff
@@ -149,9 +151,18 @@ func concatPathDirToFilePathIfRelative(basePath, filePath string) string {
 
 // processItemImages looks for image object and uploads the image when needed.
 // it processes image and supportedByImage, changing the object keys with respectively imageUrl and supportedByImageUrl after the upload
-func processItemImages(ctx context.Context, client *client.APIClient, companyID string, item *marketplace.Item, itemIDToFilePathMap map[string]string) error {
-	processImage := func(objKey, urlKey string) error {
+func processItemImages(
+	ctx context.Context,
+	client *client.APIClient,
+	companyID string,
+	item *marketplace.Item,
+	itemIDToFilePathMap map[string]string,
+) error {
+	processImage := func(objKey, urlKey string, assetType string) error {
 		localPath, err := getAndValidateImageLocalPath(item, objKey, urlKey)
+		if assetType != imageAssetType && assetType != supportedByImageAssetType {
+			return fmt.Errorf("unknown asset type %s", assetType)
```

Comment:
```suggestion
			return fmt.Errorf("unknown asset type: %s", assetType)
```
should we create a `ErrUnknownAssetType`?

Reply:
I agree 🚀